math.go

```diff
@@ -378,6 +379,78 @@ var (
 			}
 			return valueNode(nil, "avg", Null, nil), nil
 		},
+		"b64decode_no_padding": func(node *Node) (result *Node, err error) {
+			if node.IsString() {
+				if _, err := node.GetString(); err != nil {
+					return nil, err
+				} else {
+					sourceString, _ := node.GetString()
```

Comment:
Here and all parts below, this part can be replaced with:
```suggestion
				if sourceString, err := node.GetString(); err != nil {
					return nil, err
				} else {
```